joomla/joomla-cms
# [5.1] Add main region on error page

Pull Request related to #42652 .
Read comment https://github.com/joomla/joomla-cms/pull/42652#discussion_r1452413376
Now whe are changing the semantic.

### Summary of Changes
Add `<main>` on `error.php` template page.
Also when there is a module in the `error-*` position

### Testing Instructions

Case 1: Go on page error and see the source
Case 2: Add module to `error-403` or `error-404` position. Go on page error and see the source

### Actual result BEFORE applying this Pull Request
Case 1 and Case 2: There is no `<main>` element


### Expected result AFTER applying this Pull Request
Case 1: Now the error message is inside the `<main>` element
Case 2:  Now the module is inside the `<main>` element



### Link to documentations
Please select:


Thanks to @brianteeman and @richard67 for the brainstorming ;)
